File: testprojects/c#/benchmark/domain/locationbased/foursquare/Badges.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.infrastructure.socialmedia.locationbased.foursquare;

namespace CSharpBenchmark.domain.locationbased.foursquare
{
    //Functional requirement 3.2
    //Test case 118: Class domain.locationbased.foursquare.Badges is not allowed to use class infrastructure.socialmedia.locationbased.foursquare.BadgesDAO 
    //Result: FALSE
    public class Badges
    {
        public Badges()
        {
            //FR5.1
            Console.WriteLine(BadgesDAO.getAllBadges());
        }
    }
}

File: testresources/java/benchmark_2012/domain/locationbased/latitude/Badges.java
package domain.locationbased.latitude;

import infrastructure.socialmedia.locationbased.latitude.BadgesDAO;

//Functional requirement 3.2
//Test case 117: Class domain.locationbased.latitude.Badges is not allowed to use class infrastructure.socialmedia.locationbased.foursquare.BadgesDAO 
//Result: TRUE
public class Badges {
	public Badges(){
		//FR5.1
		System.out.println(BadgesDAO.getAllBadges());
	}
}